### PR TITLE
Update policy and FAQ docs for Apollo launch

### PR DESCRIPTION
**Pull request by awfuchs**
Update the spam FAQ doc

[Developer program policies](https://deploy-preview-916--developer-chrome-com.netlify.app/docs/webstore/program_policies/)
[Deceptive installation tactics FAQ](https://deploy-preview-916--developer-chrome-com.netlify.app/docs/webstore/deceptive_installation_tactics/)
[Spam FAQ](https://deploy-preview-916--developer-chrome-com.netlify.app/docs/webstore/spam_faq/)

### CODE REVIEW

File: site/en/docs/webstore/spam-faq/index.md [hunk 1]
```diff
@@ -2,156 +2,202 @@
...
 
-- Product ABC - UK
-- Product ABC - Germany
+Yes. Extensions that provide duplicate experiences—and which differ only in localization—violate the
+policy. Developers should implement any language-specific code within a single extension.
 
-Developers should implement any language-specific code within a single extension.
 
-## **Keyword spam** {: #keyword-spam }
+<h2>Keyword spam</h2>
```
Comment (dotproto) [inline site/en/docs/webstore/spam-faq/index.md:142]:
```suggestion
## Keyword spam {: #keyword-spam }
```

There doesn't seem to be a benefit to using an H2 over the Markdown equivalent.

File: site/en/docs/webstore/spam-faq/index.md [hunk 2]
```diff
@@ -2,156 +2,202 @@
...
 
-### How many sites/keywords/brands can be listed in the metadata? {: #keyword-repitition }
+### How many sites/keywords/brands can be listed in the metadata?
```
Comment (dotproto) [inline site/en/docs/webstore/spam-faq/index.md:144]:
```suggestion
### How many sites/keywords/brands can be listed in the metadata? {: #keyword-repetition }
```

Re-add the ID tag. Unfortunately there was a typo in the previous tag name. The original motivation for adding these tags was to make the final URL more human readable and consistent across title changes. Compare the following:

* Explicit: https://developer.chrome.com/docs/webstore/spam-faq/#keyword-repetition
* Auto-generated: https://developer.chrome.com/docs/webstore/spam-faq/#how-many-siteskeywordsbrands-can-be-listed-in-the-metadata

File: site/en/docs/webstore/spam-faq/index.md [hunk 3]
```diff
@@ -2,156 +2,202 @@
...
 
 At most five instances of a keyword. Keywords must be relevant to the extension's purpose. Do not
 repeat brand names in the description. When listing supported websites or brands in the description,
 do not list more than five. To provide a longer list of brands or websites, provide a link that
 users can refer to or embed the list in one of the extension's promotional screenshots.
 
-### Can the extension description contain information about additional topics related to the extension? {: #unrelated-information }
+
+### Can the extension description contain information about additional topics related to the extension?
```
Comment (dotproto) [inline site/en/docs/webstore/spam-faq/index.md:152]:
```suggestion
### Can the extension description contain information about additional topics related to the extension? {: #unrelated-information }
```

Remove extra newline and re-add the ID tag. Removing the ID end may break existing links.

File: site/en/docs/webstore/spam-faq/index.md [hunk 4]
```diff
@@ -2,156 +2,202 @@
...
 
 No. Don't provide information that is not directly relevant to the extension. For example, sports
 team wallpaper should not include team stats and history in the description of the wallpaper
 extension. Similarly, an extension targeted at a particular industry should not include general
-information about that industry in its description.
+information about that industry in its description. 
+
```
Comment (dotproto) [inline site/en/docs/webstore/spam-faq/index.md:159]:
```suggestion
```

Remove extra newline.

File: site/en/docs/webstore/spam-faq/index.md [hunk 5]
```diff
@@ -2,156 +2,202 @@
...
 
-### What is the maximum number of times the extension's primary purpose can be mentioned in the description? {: #purpose-repitition }
+### What is the maximum number of times the extension's primary purpose can be mentioned in the description?
```
Comment (dotproto) [inline site/en/docs/webstore/spam-faq/index.md:160]:
```suggestion
### What is the maximum number of times the extension's primary purpose can be mentioned in the description? {: #purpose-repetition }
```

Re-add the ID tag. Unfortunately there was a typo in the previous tag name.

File: site/en/docs/webstore/spam-faq/index.md [hunk 6]
```diff
@@ -2,156 +2,202 @@
...
 
 Do not mention one word or phrase repeatedly in the description even if it is the primary purpose of
 the extension. For example, an extension that provides puzzles should not call out the word _puzzle_
 more than five times in the description.
 
-### Can the extension's short description be repeated in the extension's description? {: #short-description-repetition }
+
+### Can the extension's short description be repeated in the extension's description?
```
Comment (dotproto) [inline site/en/docs/webstore/spam-faq/index.md:167]:
```suggestion
### Can the extension's short description be repeated in the extension's description? {: #short-description-repetition }
```

Remove extra newline and re-add the ID tag.

File: site/en/docs/webstore/spam-faq/index.md [hunk 7]
```diff
@@ -2,156 +2,202 @@
...
 
 No, the short description included in the extension's manifest.json should not be repeated in its
 Chrome Web Store's description. This string is automatically included above the description in the
 Chrome Web Store listing.
 
-## **User ratings, reviews, and installs** {: #ratings-and-reviews }
```
Comment (dotproto) [inline site/en/docs/webstore/spam-faq/index.md:173]:
```suggestion
```

Remove extra newline.

File: site/en/docs/webstore/spam-faq/index.md [hunk 8]
```diff
@@ -2,156 +2,202 @@
...
 
-### What does it mean to manipulate placement? {: #manipulate-placement }
+<h2>User ratings, reviews, and installs</h2>
+
+### What does it mean to manipulate placement?
```
Comment (dotproto) [inline site/en/docs/webstore/spam-faq/index.md:176]:
```suggestion
## User ratings, reviews, and installs {: #ratings-and-reviews }

### What does it mean to manipulate placement? {: #manipulate-placement }
```

File: site/en/docs/webstore/spam-faq/index.md [hunk 9]
```diff
@@ -2,156 +2,202 @@
...
 
 Manipulating placement refers to artificially manipulating how the Chrome Web Store orders and
 displays an extension. Such practices are expressly forbidden.
 
-### Can I provide incentives for users to download my extension? {: #download-incentives }
+
+### Can I provide incentives for users to download my extension?
```
Comment (dotproto) [inline site/en/docs/webstore/spam-faq/index.md:182]:
```suggestion
### Can I provide incentives for users to download my extension? {: #download-incentives }
```

File: site/en/docs/webstore/spam-faq/index.md [hunk 10]
```diff
@@ -2,156 +2,202 @@
...
 
 No. The offer of incentives to download a Chrome Web Store extension is not allowed. For example, a
 game should not award in-game bonuses for installing an extension. The features of your extension
 should be incentive enough.
 
-### Can I review or rate my own extension? {: #self-review }
+
+### Can I review or rate my own extension?
```
Comment (dotproto) [inline site/en/docs/webstore/spam-faq/index.md:189]:
```suggestion
### Can I review or rate my own extension? {: #self-review }
```

File: site/en/docs/webstore/spam-faq/index.md [hunk 11]
```diff
@@ -2,156 +2,202 @@
...
 
 No. Submitting a rating or review of your own extension, using your own account or other accounts,
 may be interpreted as an attempt to manipulate the extension's placement in the Chrome Web Store.
 This also includes ratings and reviews coming from other developers or users that are affiliated
 with the publisher.
 
-## **Extension functionality** {: #functionality }
```
Comment (dotproto) [inline site/en/docs/webstore/spam-faq/index.md:196]:
```suggestion
```

Remove extra newline.

File: site/en/docs/webstore/spam-faq/index.md [hunk 12]
```diff
@@ -2,156 +2,202 @@
...
 
-### Can I publish an extension that directs users to my app? {: #app-promotion }
+<h2>Extension functionality</h2>
+
+### Can I publish an extension that directs users to my app?
```
Comment (dotproto) [inline site/en/docs/webstore/spam-faq/index.md:199]:
```suggestion
## Extension functionality {: #functionality }

### Can I publish an extension that directs users to my app? {: #app-promotion }
```

File: site/en/docs/webstore/spam-faq/index.md [hunk 13]
```diff
@@ -2,156 +2,202 @@
...
+
+### Can developers appeal a repetitive content violation?
 
 You can appeal a repetitive content violation if you think it was flagged in error. To appeal, see
 the instructions included in the notification email—this is sent to the email address listed under
 your CWS developer account.
 
-### How can developers consolidate their user base? {: #consolidation }
+
+### How can developers consolidate their user base?
```
Comment (dotproto) [inline site/en/docs/webstore/spam-faq/index.md:109]:
```suggestion
### How can developers consolidate their user base? {: #consolidation }
```

File: site/en/docs/webstore/spam-faq/index.md [hunk 14]
```diff
@@ -2,156 +2,202 @@
...
 
-### Will any of the repetitive extensions be retained? {: #retention }
+
+* Choose one instance to retain
+* Update the other instances to tell your users how to migrate to the retained instance
+* Publish these updates
+* Remove all other instances from the Chrome Web Store in order to avoid enforcement action
+  against your extensions. 
+
+
```
Comment (dotproto) [inline site/en/docs/webstore/spam-faq/index.md:125]:
```suggestion
```

Remove extra newline.

**Review by dotproto — CHANGES_REQUESTED** [review]
Looks like the doc to markdown tool may have introduced a number of unexpected changes.

File: site/en/docs/webstore/spam-faq/index.md [hunk 15]
```diff
@@ -2,156 +2,202 @@
...
+### Will any of the repetitive extensions be retained?
 
 No. On August 27, 2020, the Chrome Web Store will begin removing repetitively submitted extensions.
 None of them will be retained. This is why you should make sure to bring your listings into
 compliance and remove duplicates before August 27, 2020.
 
-### Are extensions that are language specific (differing only in localization) considered duplicate? {: #localization }
+Will extensions that provide duplicate experiences, but that are geo-focused & localised (language
+specific) be considered a violation of repetitive content policy? For example:
```
[the remark this answers is not in the document]
Comment (awfuchs) [reply site/en/docs/webstore/spam-faq/index.md:122]: Done.

**Review by dotproto — APPROVED** [review]
LGTM

**Comment by dotproto** [comment]
@googlebot I consent

**Pull request merged**